packages/components/src/internal/components/user/APIKeysPanel.tsx

```diff
@@ -31,6 +31,14 @@ import { GridPanel } from '../../../public/QueryModel/GridPanel';
 import { Modal } from '../../Modal';
 import { HelpLink } from '../../util/helpLinks';
 
+const API_KEYS_QUERY_HREF = ActionURL.buildURL('query', 'executeQuery.view', '/', {
+    schemaName: 'core',
+    queryName: 'APIKeys',
+});
+const CUSTOMIZE_SITE_HREF = ActionURL.buildURL('admin', 'customizeSite.view', '/');
+const API_KEYS_DOCS_HREF = 'https://www.labkey.org/Documentation/wiki-page.view?name=apiKey#usage';
```

Comment:
Consider using the `getHelpLink()` utility to generate these documentation links:

```ts
// getHelpLink() defaults to referrer=inPage which is what we want for both of these
const API_KEYS_DOCS_HREF = getHelpLink('apiKey#usage', undefined, true);
const CLIENT_APIS_HREF = getHelpLink('viewApis', undefined, true);
```